bundles/org.openhab.binding.ecobee/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -315,6 +319,26 @@
 		<label>Desired Cool Range High</label>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<channel-type id="actualAQAccuracy" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
```

Comment:
This channel has no unit set when updating its state, so it should be linked to an item of type "Number", not "Number:Dimensionless".
README should be also updated.

Reply:
Fixed